### PR TITLE
ABE-653 - Create operations controller.

### PR DESCRIPTION
**Pull request by michaelleahy**
@lookinginstereo @joshhaynes @jsatria 

A few things are commented out, but they'll be active very soon.  Trying to keep the PRs small.

### CODE REVIEW

**Comment by michaelleahy** [comment]
retest this please

File: service/src/main/scala/com/rallyhealth/donations/util/DonationsServiceModules.scala [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+class InternalModules(config: DonationsConfig) extends AbstractModule with ScalaModule {
+  override protected def configure() = {
+    bind[DonationsConfig].toInstance(config)
+
+  }
+}
+
+class UserServiceModules(config: DonationsConfig) extends AbstractModule with ScalaModule {
+  override protected def configure() = {
+//    if (config.userServiceEnabled) {
```
Comment (joshhaynes) [inline service/src/main/scala/com/rallyhealth/donations/util/DonationsServiceModules.scala]: Why are these commented out?

File: service/src/test/scala/com/rallyhealth/donations/util/DonationsServiceModulesSpec.scala [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  describe("The Donations Service modules"){
+
+    it("only need an execution context binding") {
+      val alpha = doit(basicConfig)
+      val beta = alpha
+      assert(alpha === beta)
+    }
+  }
+
+//  describe("The Admin controller module") {
```
Comment (joshhaynes) [inline service/src/test/scala/com/rallyhealth/donations/util/DonationsServiceModulesSpec.scala]: Why is this commented out?

File: web/test/com/rallyhealth/donations/util/DonationsWebDISpec.scala [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  describe("The Donations Web DI injector"){
+
+    it("have the same instance when constructed multiple times") {
+        val alpha = DonationsWebDI.injector
+        val beta = DonationsWebDI.injector
+        assert(alpha === beta)
+    }
+  }
+//
+//  describe("The Admin controller module") {
```
Comment (joshhaynes) [inline web/test/com/rallyhealth/donations/util/DonationsWebDISpec.scala]: Same question

File: service/src/main/scala/com/rallyhealth/donations/util/DonationsServiceModules.scala (hunk 1 above)
Comment (michaelleahy) [inline service/src/main/scala/com/rallyhealth/donations/util/DonationsServiceModules.scala]: Did you read the description of the PR?

**Comment by joshhaynes** [comment]
Strong armed.  LGTM

**Comment by michaelleahy** [comment]
:angry:

**Comment by Ackertron** [comment]
retest this please

**Comment by Ackertron** [comment]
retest this please

**Comment by michaelleahy** [comment]
@lookinginstereo / @jsatria any other thoughts?

**Comment by michaelleahy** [comment]
1LZF

**Pull request merged**